src/shared/components/FormComponents/EditorController/EditorController.const.ts

```diff
@@ -0,0 +1 @@
+export const EDITOR_CONTROLLER_DEBOUNCE_VALUE = 500;
```

Comment:
should not we use common debounce value for all inputs from shared constants? why we need to keep a separate value for editor?

Reply:
done